i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md

```diff
@@ -5,49 +5,89 @@ title: WD NAS SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Contenu du Pack
+## Contenu du pack
 
 ### Modèles
 
-Le Pack Centreon **WD NAS SNMP** apporte un modèle d'hôte :
+Le connecteur de supervision **WD NAS SNMP** est compatible avec les équipements WD NAS suivants :
+* EX2
+* EX2 Ultra
+* EX4100
+* PR2100
+* PR4100
 
-* HW-Storage-Wd-Nas-SNMP-custom
+Les autres équipements peuvent être supportés mais sans garanties.
```

Comment:
J'ai ajouté les modèles d'équipement qui étaient supportés pour plus de visibilité pour l'utilisateur (sachant que la PR Communautaire du ticket https://centreon.atlassian.net/browse/CTOR-397 se rapporte justement au fait d'ajouter en plus le mode EX4100 (noté ici). Donc on ne pourra merge cette doc qu'une fois la PR sortie (on peut rebase sur la branche de Release 24.10 si besoin)